streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -156,14 +228,16 @@ public void shouldRestoreStateFromSourceTopic(final boolean stateUpdaterEnabled)
         final Properties props = props(stateUpdaterEnabled);
         props.put(StreamsConfig.TOPOLOGY_OPTIMIZATION_CONFIG, StreamsConfig.OPTIMIZE);
 
-        // restoring from 1000 to 4000 (committed), and then process from 4000 to 5000 on each of the two partitions
+        // restoring from 1000 to 4000 (committed), and then process from 4000 to 5000
```

Comment:
The newlines introduced below till line 496 seems to be auto from IDE, could you revert them if they are not intentional?